test/Base_Tests/src/Runtime/State_Spec.enso

```diff
@@ -48,6 +50,14 @@ add_specs suite_builder = suite_builder.group "State" group_builder->
 
         fourty_two . should_equal 42
 
+    group_builder.specify "State from Enso to Java to Enso" <|
+        s = State.run Test 42 <|
+            obtain_state delta =
+                State.get Test + delta
+
+            CallbackHelper.runCallbackInt obtain_state -1
+        s . should_equal 41
```

Comment:
This is great.

The #7117 ticket was also about Contexts not being propagated in the same manner.

Does the fix to the State also fix that?
- If yes, can we add an analogous test case to show it?
- If not, will we not close the ticket OR create a separate one to track the Contexts issue?

Reply:
Looking at the code, the contexts seem to already be a thread local, so maybe it was fixed earlier.

But after a search I could not find a test for the Contexts. Can we add a related test, to show that the context also are working correctly with regards to the polyglot calls? Then we could close #7117 at peace, knowing that it has been resolved in whole.

Reply:
- Yes, it has been fixed before
- probably by #11173
- Verified in 800633c358